src/components/createTodo/index.tsx

```diff
@@ -0,0 +1,87 @@
+// import { zodResolver } from '@hookform/resolvers/zod';
+// import { useForm } from 'react-hook-form';
+import { useRef, useState } from 'react';
+
+import AddIcon from '@/assets/icons/AddIcon';
+import Dialog from '@/components/dialog';
+import useKabanaStore from '@/stores/store';
+
+import Tag from '../tag';
+import colorList from './colorList';
+const CreateTodo = () => {
+  const modalIsOpen = useKabanaStore((state) => state.createTodo);
+  const toggleModal = useKabanaStore((state) => state.toggleCreateTodo);
+  const [tagList, setTagList] = useState<string[]>([]);
+  const [addedColors, setAddedColors] = useState<string[]>([]);
+  const tagRef = useRef<HTMLInputElement>(null);
+
+  const createRandomNumber = () => {
+    const result = Math.floor(Math.random() * colorList.length);
+    return result;
+  };
+
+  const createDeleteTags = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === 'Enter') {
+      const randomNum = createRandomNumber();
+      if (tagList.includes(e.currentTarget.value)) {
+        e.currentTarget.value = '';
```

Comment:
아래처럼 return을 추가해야 할 것 같습니다. 태그 중복일때 아래코드가 계속 실행되는걸 막을수 있을거 같아요. 
```suggestion
if (tagList.includes(e.currentTarget.value)) {
  e.currentTarget.value = '';
  return; 
}
```

Reply:
넵 수정하겠습니다!